src/Sparrow/project.json

```diff
@@ -20,10 +20,17 @@
         "optimize": true,
         "define": [ "MEM_GUARD" ]
       }
+    },
+    "Validate": {
+      "buildOptions": {
+        "optimize": true,
+        "define": [ "MEM_GUARD", "MEM_GUARD_STACK" ]
+      }
     }
   },
 
   "dependencies": {
+    "DotNetCross.Memory.Unsafe": "0.2.3.4",
```

Comment:
In this case a lot. This guy figured out using raw IL how to generate a repl move operation. For small size byte arrays this can be very fast. Also in 1.2 this JIT intrinsic has been optimized even further. Having said that the complexity of our copy is too big to be inlined here so the actual cost is dominated by the call overhead